File: src/types/review.ts
export interface ReviewEntry {
  id: string;
  reviewerName: string;
  reviewerInitials: string;
  countryCode: string;
  country: string;
  rating: number;
  title: string;
  comment: string;
  tags: string[];
  service: string;
  dateValue: string;
  relativeDate: string;
}

File: backend/app/schemas/review.py
from pydantic import BaseModel


class ReviewEntry(BaseModel):
    id: str
    reviewerName: str
    reviewerInitials: str
    country: str
    countryCode: str
    rating: float
    title: str
    comment: str
    tags: list[str]
    service: str
    dateValue: str
    relativeDate: str
